packages/common/package.json

```diff
@@ -1,45 +1,45 @@
 {
-    "name": "@map-colonies/detiler-common",
-    "version": "1.3.0",
-    "description": "detiler common types",
-    "main": "./dist/index.js",
-    "types": "./dist/index.d.ts",
-    "scripts": {
-        "test:unit": "jest --config=./tests/configurations/unit/jest.config.js",
-        "format": "prettier --check . --ignore-path ../../.prettierignore",
-        "format:fix": "prettier --write . --ignore-path ../../.prettierignore",
-        "prelint:fix": "npm run format:fix",
-        "prelint": "npm run format",
-        "lint": "eslint --config ../../.eslintrc.json .",
-        "lint:fix": "eslint --config ../../.eslintrc.json --fix .",
-        "prebuild": "npm run clean",
-        "build": "tsc --project tsconfig.build.json",
-        "clean": "rimraf dist",
-        "prepack": "npm run build"
-    },
-    "repository": {
-        "type": "git",
-        "url": ""
-    },
-    "author": "MapColonies",
-    "license": "ISC",
-    "files": [
-        "dist"
-    ],
-    "devDependencies": {
-        "@map-colonies/eslint-config": "^4.0.0",
-        "@map-colonies/prettier-config": "0.0.1",
-        "@types/geojson": "7946.0.14",
-        "@types/jest": "^29.5.2",
-        "eslint": "^8.56.0",
-        "jest": "^29.5.0",
-        "jest-create-mock-instance": "^2.0.0",
-        "jest-html-reporters": "^3.1.4",
-        "prettier": "^2.8.8",
-        "pretty-quick": "^3.1.3",
-        "rimraf": "^5.0.5",
-        "ts-jest": "^29.1.1",
-        "ts-node": "^10.9.2",
-        "typescript": "^5.3.3"
-    }
+  "name": "@map-colonies/detiler-common",
+  "version": "1.3.0",
+  "description": "detiler common types",
+  "main": "./dist/index.js",
+  "types": "./dist/index.d.ts",
+  "scripts": {
+    "test:unit": "jest --config=./tests/configurations/unit/jest.config.js",
+    "format": "prettier --check . --ignore-path ../../.prettierignore",
+    "format:fix": "prettier --write . --ignore-path ../../.prettierignore",
+    "prelint:fix": "npm run format:fix",
+    "prelint": "npm run format",
+    "lint": "eslint --config ../../.eslintrc.json .",
+    "lint:fix": "eslint --config ../../.eslintrc.json --fix .",
+    "prebuild": "npm run clean",
+    "build": "tsc --project tsconfig.build.json",
+    "clean": "rimraf dist",
+    "prepack": "npm run build"
+  },
+  "repository": {
+    "type": "git",
+    "url": ""
+  },
+  "author": "MapColonies",
+  "license": "ISC",
+  "files": [
+    "dist"
+  ],
+  "devDependencies": {
+    "@map-colonies/eslint-config": "^4.0.0",
+    "@map-colonies/prettier-config": "0.0.1",
+    "@types/geojson": "7946.0.14",
+    "@types/jest": "^29.5.2",
+    "eslint": "^8.56.0",
+    "jest": "^29.5.0",
+    "jest-create-mock-instance": "^2.0.0",
+    "jest-html-reporters": "^3.1.4",
```

Comment:
update `eslint` and `jest` deps, this is also true for all other monorepo packages.